src/doc/unstable-book/src/language-features/abi-thiscall.md

```diff
@@ -0,0 +1,7 @@
+# `abi_thiscall`
+
+The tracking issue for this feature is: none.
```

Comment:
Can we create a tracking issue for this? I think it'd be fine to have one tracking issue covering all the various unstable ABIs. It'd also be nice to actually put a few words in there about what the purpose of this ABI is. If we're going to have a book we might as well write something in it. =)